frontend/sw360-portlet/src/main/webapp/html/search/view.jsp

```diff
@@ -153,7 +153,17 @@
         result.push({
             "DT_RowId": '${doc.id}',
             "0": '${doc.type}',
-            "1": "<sw360:DisplaySearchResultLink searchResult="${doc}"/>"
+            <core_rt:choose>
+                <core_rt:when test="${doc.type.equals('project')
+                                   || doc.type.equals('component')
+                                   || doc.type.equals('release')
```

Comment:
I think something along the lines of this reply should work: http://stackoverflow.com/a/1490181